### PR TITLE
removed babel

### PR DESCRIPTION
**Pull request by alphmth**

### CODE REVIEW

File: packages/test-suite/package.json [hunk 1]
```diff
@@ -1,62 +1,50 @@
...
-    "this": "^1.1.0",
-    "xo": "^0.47.0"
-  }
+	"name": "@keyv/test-suite",
+	"version": "1.6.15",
+	"description": "Test suite for Keyv API compliancy",
+	"main": "src/index.js",
+	"type": "commonjs",
+	"scripts": {
+		"test": "xo --fix && nyc ava",
```
Comment (jaredwray) [inline packages/test-suite/package.json]: please remove the `--fix`

**Comment by jaredwray** [comment]
@alphmth - can you validate what this error is?

https://capture.dropbox.com/Vj41wmBNFjZ0FGM4?src=ss

**Comment by jaredwray** [comment]
@alphmth - with these changes will this affect all the 3rd party people using the test-suite? Is this a breaking change?

**Comment by bilalsha** [comment]
> 

This is a linting error. I will fix it

**Comment by alphmth** [comment]
> @alphmth - can you validate what this error is?
> 
> https://capture.dropbox.com/Vj41wmBNFjZ0FGM4?src=ss

its ESLint linting warning I have resolved it

**Comment by alphmth** [comment]
> @alphmth - with these changes will this affect all the 3rd party people using the test-suite? Is this a breaking change?

I have refactored some code and it should not affect 3rd party people using test-suite package

**Pull request merged**